### PR TITLE
OPS-3946 Fixing deprecation warnings with PHP 5.5

### PR DESCRIPTION
**Pull request by gleseur**
Check the app is loading fine

### CODE REVIEW

File: lib/util/sfToolkit.class.php [hunk 1]
```diff
@@ -355,7 +355,14 @@ public static function replaceConstants($value)
    */
   public static function pregtr($search, $replacePairs)
   {
-    return preg_replace(array_keys($replacePairs), array_values($replacePairs), $search);
+    forEach($replacePairs as $key => $val) {
+      if (substr($key, -1) === 'e'){
+        $search = preg_replace_callback(substr_replace($key, "", -1), $val, $search);
```
Comment (olasfar) [inline lib/util/sfToolkit.class.php]: shouldn't we check that $val is_callable, do we know how PHP will behave if $val isn't a function ?

**Comment by olasfar** [comment]
Unit tests ?

File: lib/util/sfToolkit.class.php (hunk 1 above)
Comment (gleseur) [inline lib/util/sfToolkit.class.php]: not well probably, but it also fails when leaving it the old way. So it's about the same.
I think what really matters here is checking usage of this function. And it's used only in a few places.

**Comment by gleseur** [comment]
@olasfar About tests, we don't run any on symfony. But I'll add some on work4us side about sfInflector::camelize
as it's clear what it should be doing.

**Pull request merged**